Applications/CMC/test/testCMCTwoMusclesOnBlock.cpp

```diff
@@ -1,141 +0,0 @@
- * distributed under the License is distributed on an "AS IS" BASIS,          *
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.   *
- * See the License for the specific language governing permissions and        *
- * limitations under the License.                                             *
- * -------------------------------------------------------------------------- */
-
-// INCLUDE
-#include <OpenSim/Simulation/Model/Model.h>
-#include <OpenSim/Simulation/Model/AnalysisSet.h>
-#include <OpenSim/Tools/CMCTool.h>
-#include <OpenSim/Tools/ForwardTool.h>
-#include <OpenSim/Auxiliary/auxiliaryTestFunctions.h>
-
-using namespace OpenSim;
-using namespace std;
-
-void testTwoMusclesOnBlock();
-
-int main() {
-
-    SimTK::Array_<std::string> failures;
-
-    try {testTwoMusclesOnBlock();}
-    catch (const std::exception& e)
-        {  cout << e.what() <<endl; failures.push_back("testTwoMusclesOnBlock"); }
-
-    // redo with the Millard2012EquilibriumMuscle 
-    Object::renameType("Thelen2003Muscle", "Millard2012EquilibriumMuscle");
-
-    try {testTwoMusclesOnBlock();}
-    catch (const std::exception& e)
-        {   cout << e.what() <<endl; 
-            failures.push_back("testTwoMusclesOnBlock_Millard"); }
-    
-    if (!failures.empty()) {
-        cout << "Done, with failure(s): " << failures << endl;
-        return 1;
-    }
-
-    cout << "Done" << endl;
-
-    return 0;
-}
-
-void testSingleRigidTendonMuscle() {
-    cout << "\n******************************************************************" << endl;
-    cout << "*                   testSingleRigidTendonMuscle                  *" << endl;
-    cout << "******************************************************************\n" << endl;
-
-    Model model("block_hanging_RigidTendonMuscle.osim");
-    Model* modelCopy = model.clone();
-    modelCopy->setup();
-    ASSERT(model == *modelCopy);
-
-    ForwardTool forward("block_hanging_from_muscle_Setup_Forward.xml");
-    forward.setModel(model);
-    forward.run();
-
-    // Use copy of the model because forward adds a ControlSetController to the model and the controls from CMC
-    // are added in with those "feedforward" controls. Instead we want to verify that CMC can compute these 
-    // same controls
-    CMCTool cmc("block_hanging_from_muscle_Setup_CMC.xml");
-    cmc.setModel(*modelCopy);
-    cmc.run();
-
-    Storage fwd_result("block_hanging_from_muscle_ForwardResults/block_hanging_from_muscle_states.sto");
-    Storage cmc_result("block_hanging_from_muscle_ResultsCMC/block_hanging_from_muscle_states.sto");
-
-    // Tolerance of 2mm or position error and 2mm/s translational velocity of the block
-    CHECK_STORAGE_AGAINST_STANDARD(cmc_result, fwd_result, Array<double>(0.0025, 4), __FILE__, __LINE__, "testSingleRigidTendonMuscle failed");
-    
-    cout << "testSingleRigidTendonMuscle passed\n" << endl;
-}
-
-
-void testSingleMillardRigidTendonMuscle() {
```

Comment:
Oops; I think it was my fault that these methods were duplicated! Thanks for removing them.